.env.test

```diff
@@ -0,0 +1 @@
+API_ENABLED: true
```

Comment:
oooh we have some other env vars in `rails_helper` - perhaps move them into here?